src/neynar-api/v2/openapi-farcaster/apis/user-api.ts

```diff
@@ -975,6 +1054,20 @@ export class UserApi extends BaseAPI {
         return UserApiFp(this.configuration).lookupUserByCustodyAddress(apiKey, custodyAddress, options).then((request) => request(this.axios, this.basePath));
     }
 
+    /**
+     * Fetches power users based on Warpcast power badges
+     * @summary Fetch power users
+     * @param {string} apiKey API key required for authentication.
+     * @param {number} [limit] Number of power users to fetch, max 100
+     * @param {string} [cursor] Pagination cursor.
+     * @param {*} [options] Override http request option.
+     * @throws {RequiredError}
+     * @memberof UserApi
+     */
+    public powerUsers(apiKey: string, limit?: number, cursor?: string, options?: AxiosRequestConfig) {
```

Comment:
Need to add the wrapper methods with name `fetchPowerUsers` in `src/neynar-api/v2/client.ts` and `src/neynar-api/neynar-api-client.ts`

Reply:
While writing the JSDocs for these methods please follow the following guidelines 
https://www.notion.so/neynar/Guidelines-for-chatGPT-to-create-docString-for-generated-Code-9b0592f0c173442685585423ad073149